app/lib/notifications.server.ts

```diff
@@ -31,18 +31,22 @@ const initializeTransport = () => {
 };
 
 const sendNotification = async (emailAddress: string, subject: string, text: string) => {
-  try {
-    const transport = initializeTransport();
-    logger.debug(`Sending notification to ${emailAddress}`);
-    return await transport.sendMail({
-      from: NOTIFICATIONS_EMAIL_USER,
-      to: emailAddress,
-      subject,
-      text,
-    });
-  } catch (error) {
-    logger.warn('Unable to send notification', error);
-  }
+  return new Promise<String>(async (resolve, reject) => {
```

Comment:
No, let's not wrap with additional promises.  An `async` function automatically wraps the `return` in a Promise, and you're adding yet another manual promise here.

We already have a Promise in the form of `transport.sendMail()`, which you can return directly:

```ts
const transport = initializeTransport();
logger.debug(`Sending notification to ${emailAddress}`);
return transport.sendMail(...).catch(err => logger.warn('Unable to send notification', err));
```

Doing this, you don't need `async` on the function either.